test/config.test.js

```diff
@@ -42,6 +44,220 @@ describe(`configTest: ${printPath("[test/config.test.js]")}`, function () {
+            try {
+                STExpress.init({
+                    supertokens: {
+                        connectionURI: "http://localhost:8080",
+                    },
+                    appInfo: {
+                        apiDomain: "api.supertokens.io",
+                        websiteDomain: "supertokens.io",
+                    },
+                    recipeList: [Session.init()],
+                });
+            } catch (err) {
+                if (
+                    err.type !== STExpress.Error.GENERAL_ERROR ||
+                    err.message !==
+                        "Please provide your appName inside the appInfo object when calling supertokens.init"
+                ) {
+                    throw err;
+                }
+            }
+
+            resetAll();
+        }
+
+        {
+            try {
+                STExpress.init({
+                    supertokens: {
+                        connectionURI: "http://localhost:8080",
+                    },
+                    appInfo: {
+                        apiDomain: "api.supertokens.io",
+                        appName: "SuperTokens",
+                    },
+                    recipeList: [Session.init()],
+                });
+            } catch (err) {
+                if (
+                    err.type !== STExpress.Error.GENERAL_ERROR ||
+                    err.message !==
+                        "Please provide your websiteDomain inside the appInfo object when calling supertokens.init"
+                ) {
+                    throw err;
+                }
+            }
+
+            resetAll();
+        }
+    });
+
+    // * TODO: test using zero, one and two recipe modules
+    it("test using zero, one and two recipe modules", async function () {
+        await startST();
+
+        {
+            try {
+                STExpress.init({
+                    supertokens: {
+                        connectionURI: "http://localhost:8080",
+                    },
+                    appInfo: {
+                        apiDomain: "api.supertokens.io",
+                        appName: "SuperTokens",
+                        websiteDomain: "supertokens.io",
+                    },
+                });
+            } catch (err) {
+                if (err.message !== "Please provide at least one recipe to the supertokens.init function call") {
+                    throw err;
+                }
+            }
+
+            resetAll();
+        }
+
+        {
+            STExpress.init({
+                supertokens: {
+                    connectionURI: "http://localhost:8080",
+                },
+                appInfo: {
+                    apiDomain: "api.supertokens.io",
+                    appName: "SuperTokens",
+                    websiteDomain: "supertokens.io",
+                },
+                recipeList: [Session.init()],
+            });
+
+            assert(SuperTokens.getInstanceOrThrowError().recipeModules.length === 1);
```

Comment:
Also get the session recipe instance via `Session.getInstanceOrThrowError()` (or whatever that function is called). If this function throws an error, then it's not properly working. Same for the test right below this one